### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by JacoboOssa**

### CODE REVIEW

**Review by yuluka — CHANGES_REQUESTED** [review]
tests_load_scholarship_info.py has conflicts.

File: crud/tests/tests_load_scholarship_info.py [hunk 1]
```diff
@@ -71,6 +71,8 @@ def test_post_valid_form(self):
         self.assertEqual(len(becas), 2)
         self.assertEqual(becas[0].id_estudiante, 'A00381234')
         self.assertEqual(becas[1].id_estudiante, 'A00385678')
+<<<<<<< HEAD
```
Comment (AndresPin0) [inline crud/tests/tests_load_scholarship_info.py:74]: Elimina eso puñeta

**Review by AndresPin0 — CHANGES_REQUESTED** [review]
Elimina eso puñeta

File: crud/tests/tests_load_scholarship_info.py [hunk 2]
```diff
@@ -71,6 +71,8 @@ def test_post_valid_form(self):
...
+=======
```
Comment (Dennis-Spiderman) [inline crud/tests/tests_load_scholarship_info.py:75]: Delete this please.

**Review by Dennis-Spiderman — CHANGES_REQUESTED** [review]

**Pull request merged**